.github/workflows/cicd.yml

```
@@ -0,0 +1,4 @@
+name: cicd.yml
+on:
```

actionlint (1.7.9)

string should not be empty

(syntax-check)